src/components/search/BusinessLogo.tsx

```diff
@@ -2,14 +2,17 @@ import { DisplayableBusiness } from '@futureproof/typings'
 import React from 'react'
 import { Image, StyleSheet, View } from 'react-native'
 
+export const DEFAULT_PROFILE_PATH = '../../../assets/icon.png'
```

Comment:
change DEFAULT_PROFILE_PATH to something along the lines of '@images/icon.png'

Reply:
See [above](https://github.com/CM20314-Group-2/client/pull/2#discussion_r810643157).